lib/profile/commands/configure.rb

```diff
@@ -54,7 +70,16 @@ def prompt
       end
 
       def cluster_type
-        @type ||= Type.find( prompt.select('Cluster type: ', Type.all.map { |t| t.name }) )
+        return @type if @type
+        if @options.answers
+          if @answers.key?("cluster_type")
```

Comment:
If `@answers` doesn't have the `'cluster_type'` key, `@answers.delete('cluster_type')` will return nil, meaning we can replace this conditional with:
```ruby
Type.find(@answers.delete('cluster_type') || Config.cluster_type)
```